### PR TITLE
Add Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by yichifauzi**
Add Traditional Chinese localization

### CODE REVIEW

File: src/main/resources/assets/aswitch/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "key.autoswitch.toggle": "AutoSwitch 切換鍵",
+  "key.autoswitch.toggle_mowing": "除草保護切換鍵",
+  "msg.autoswitch.mow_false": "戰鬥時除草：停用",
+  "msg.autoswitch.mow_true": "戰鬥時除草：啟用",
+  "msg.autoswitch.toggle_false": "AutoSwitch：停用",
+  "msg.autoswitch.toggle_true": "AutoSwitch：啟用",
+  "msg.autoswitch.config.1": "AutoSwitch 設定檔位於 config 資料夾中。",
+  "msg.autoswitch.config.2": "請勿開啟或反編譯 AutoSwitch 來更改它們！",
```
Comment (notlin4) [inline src/main/resources/assets/aswitch/lang/zh_tw.json:9]:
```suggestion
  "msg.autoswitch.config.2": "請勿開啟或反編譯 AutoSwitch 來修改設定！",
```

File: src/main/resources/assets/aswitch/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "msg.autoswitch.config.3": "快速前往該資料夾的方法是：前往「選項」，然後「開啟資源包資料夾」。",
```
Comment (notlin4) [inline src/main/resources/assets/aswitch/lang/zh_tw.json:10]:
```suggestion
  "msg.autoswitch.config.3": "快速前往該資料夾的方法是：前往「選項」，然後按下「開啟資源包資料夾」。",
```

File: src/main/resources/assets/aswitch/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "msg.autoswitch.config.4": "然後向上移動一層，使您位於 minecraft 資料夾中。您將看到 config 資料夾。",
```
Comment (notlin4) [inline src/main/resources/assets/aswitch/lang/zh_tw.json:11]:
```suggestion
  "msg.autoswitch.config.4": "接著回到上一層，你就會在你的 Minecraft 資料夾中看到 config 資料夾。",
```

File: src/main/resources/assets/aswitch/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "msg.autoswitch.config.5": "config 資料夾中將包含您的設定檔。",
```
Comment (notlin4) [inline src/main/resources/assets/aswitch/lang/zh_tw.json:12]:
```suggestion
  "msg.autoswitch.config.5": "你的設定檔就會在 config 資料夾中。",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by dexman545** [comment]
Thanks for the contribution!

**Pull request merged**